src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,11 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (User i : users) {
+            if (i.getEmail().equals(email)) {
+                return i;
+            }
+        }
```

Comment:
```suggestion
        for (User user : users) {
            if (i.getEmail().equals(email)) {
                return i;
            }
        }
```
use understandable names